README.md

```diff
@@ -1,15 +1,74 @@
 # Ionic Langchain
 
-Pre-release. Do not use this.
+Ionic Langchain provides a wrapper around the Ionic Commerce's SDK for use as a `Tool` in a custom Langchain agent.  This tool will enable e-commerce for your agent, allowing your users to ask for product recommendations and purchase products through the agent chat interface.
 
 ## Installation
 
-Pre-release. Do not use this.
+This tool requires at least `langchain@0.0.350` and can work with any greater patch release the `0.0.x` series.
```

Comment:
I've never written a public package; not sure about python version compatibility or how we should be managing that.  I figure we don't need that right away but IMO worth figuring out.

Reply:
We should only support the latest LTS for the start. 

When someone pays us money and needs a specific version that's another story imo.